src/Demo.java

```diff
@@ -0,0 +1,28 @@
+public class ComplexMethod {
+
+    public String intToEnglishValue(int i) {
+        if (i == 1) {
+            return "One";
+        }
+        if (i == 2) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 2 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS109)